vcr/vcr.go

```diff
@@ -227,7 +227,7 @@ func (c *vcr) search(query concept.Query, allowUntrusted bool, resolveTime *time
 				return nil, errors.Wrap(err, "unable to parse credential from db")
 			}
 
-			if err = c.Validate(foundCredential, allowUntrusted, resolveTime); err == nil {
+			if err = c.Validate(foundCredential, allowUntrusted, false, resolveTime); err == nil {
```

Comment:
just curious, what would be the use case for searching for VCs with invalid signatures v.s. VCs with valid signatures?

Reply:
VCs signatures are checked before a VC is written to the store. So the store only has valid VCs.
Checking them on return slows things down (and makes the unit tests a lot harder)

Reply:
I see in VCR's `Ambassador`:
> // vcCallback gets called when new Verifiable Credentials are received by the network. All checks on the signature are already performed.

I assume the godoc means the network layer checks the signature. That's correct, for the TX signature. But there's also a `proof` in our Nuts VCs with a `jws` field, which isn't checked?

Update: just checked, VCR's `store.StoreCredential` verifies the VC signature. So the ambassador's comment is a bit misleading ("signatures" being plural, while just meaning "TX's signature"). Maybe we should change the comment to "the transaction's signature has already been verified".